node/network/protocol/Cargo.toml

```diff
@@ -12,3 +12,7 @@ polkadot-node-jaeger = { path = "../../jaeger" }
 parity-scale-codec = { version = "1.3.6", default-features = false, features = ["derive"] }
 sc-network = { git = "https://github.com/paritytech/substrate", branch = "master" }
 strum = { version = "0.20", features = ["derive"] }
+thiserror = "1.0.23"
+
+[target.'cfg(not(target_os = "unknown"))'.dependencies]
+zstd = "0.5.0"
```

Comment:
the latest is 0.6.0, also maybe we should disable `legacy` feature enabled by default, and there is `wasm` feature as well 
https://github.com/gyscos/zstd-rs/blob/master/Cargo.toml#L38

Reply:
We can not use this. Wasmtime uses this old version and you can not use a native library in 2 different versions.

Reply:
TIL: wasmtime compresses its wasm modules cache using `zstd`.

Reply:
(the suggestion to use `wasm` feature instead of `cfg` shenanigans in code still holds)

Reply:
I think we actually don't use the wasmtime cache.

Reply:
It fails to link. Honestly I don't want to waste more time on this ***** wasm build. It took me already too much time for no gain. Having a browser node participating the normal validation network is a bad idea anyway.

Reply:
Fair enough :)